samplesheets/views_api.py

```diff
@@ -310,6 +317,188 @@ def post(self, request, *args, **kwargs):
         return Response(ret_data, status=status.HTTP_200_OK)
 
 
+class IrodsAccessTicketListAPIView(
+    IrodsAccessTicketModifyMixin, SODARAPIBaseProjectMixin, ListAPIView
```

Comment:
The `IrodsAccessTicketModifyMixin` depdendency should not be included for list or retrieve views, as the object is not modified by those views.